core/src/connection/listeners.rs

```diff
@@ -148,6 +148,8 @@ where
     Closed {
         /// The ID of the listener that closed.
         listener_id: ListenerId,
+        /// The addresses that the listener was listening on.
+        addresses: SmallVec<[Multiaddr; 4]>,
```

Comment:
This is a nit, but I'd rather go with `Vec` in order to not expose a third-party dependency in our API.
Alternatively, a custom iterator type that wraps around `smallvec::IntoIter` would work as well.

Reply:
No worries, will fix and re-spin.  Thanks.